go.mod

```diff
@@ -83,17 +119,37 @@ require (
 	github.com/juju/schema v1.0.1-0.20190814234152-1f8aaeef0989
 	github.com/juju/terms-client/v2 v2.0.0-20210422053140-27f71c100676
 	github.com/juju/testing v0.0.0-20210324180055-18c50b0c2098
+	github.com/juju/txn v0.0.0-20210302043154-251cea9e140a // indirect
 	github.com/juju/txn/v2 v2.0.0-20210407000251-11166e89894c
```

Comment:
this sort of thing concerns me, that we would depend on multiple versions of the same library, and that seems to happen a lot.

Reply:
`go mod why` is our friend here:

```
go mod why github.com/juju/txn
# github.com/juju/txn
github.com/juju/juju/scripts/juju-blobstore-cleanup
github.com/juju/blobstore/v2
github.com/juju/txn
```

Reply:
[This is still using txn v1](https://github.com/juju/blobstore/blob/v2/managedstorage.go#L24), probably needs updating tbh